packages/esbuild-plugin/package.json

```diff
@@ -1,40 +1,71 @@
 {
     "name": "@datadog/esbuild-plugin",
     "packageManager": "yarn@4.0.2",
+    "version": "0.0.13-4",
     "license": "MIT",
     "author": "Datadog",
+    "description": "Datadog ESBuild Plugin.",
+    "keywords": [
+        "datadog",
+        "esbuild",
+        "bundler",
+        "plugin",
+        "unplugin"
+    ],
+    "homepage": "https://github.com/DataDog/build-plugin#readme",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/DataDog/build-plugin",
+        "directory": "packages/esbuild-plugin"
+    },
     "main": "./dist/src/index.js",
+    "module": "./dist/src/index.mjs",
     "exports": {
         ".": "./src/index.ts",
         "./*": "./src/*.ts"
```

Comment:
Shouldn't we drop the `./*` (as it's not overridden in the `publishConfig` which means that it won't work for built code)

Reply:
Good catch.